### DISCUSSION TITLE
Features of Maya 2020 w/ UFE 1.0 vs May 2022 UFE 2.0

### DISCUSSION THREAD

**Discussion by emily-vo538** [Autodesk/maya-usd#1375] [Q&A]
Is there an official catalog of features between the two versions? Specifically for USD?

**Comment by wtelford** [comment]
The following features require UFE 2.0 

- re-parenting
- undo/redo
- duplicate
- rename
- reorder
- outliner visualization of mixed-data model, data type icons, context menus, usd display filters, selection framing
- attribute editor
- some VP2 performance enhancements
- advanced manipulation of arbitrary transform stacks
- viewport selection framing
- viewport purpose filtering
- mixed data model compute
- advance serialization
